### PR TITLE
more lxqt-files

### PR DESCRIPTION
**Pull request by Besanon**
Add initial support for new wayland session in lxqt 2.1 using labwc and kwin_wayland.

### CODE REVIEW

File: apparmor.d/groups/lxqt/lxqt-config-powermanagement [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  owner @{user_config_dirs}/lxqt/lxqt-powermanagement.conf.@{rand6} l -> @{user_config_dirs}/lxqt/#@{int},
+
+  owner /tmp/@{int} r,
+
+  @{sys}/class/backlight/ r,
+  @{sys}/devices/@{pci_bus}/**/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/* rw,
+  @{sys}/devices/@{pci_bus}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/ r,
+  @{sys}/devices/@{pci_bus}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/max_brightness r,
+  @{sys}/devices/@{pci_bus}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/bl_power r,
+  @{sys}/devices/@{pci_bus}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/actual_brightness  r,
```
Comment (nobody43) [inline apparmor.d/groups/lxqt/lxqt-config-powermanagement:33]: Addresses.

Comment (Besanon) [reply apparmor.d/groups/lxqt/lxqt-config-powermanagement:33]: Hello, could you please comment abit more than just a word. thank you

Comment (nobody43) [reply apparmor.d/groups/lxqt/lxqt-config-powermanagement:33]: Should start with `@{sys}/devices/@{pci}/`, check other examples:
`grep -r \{pci\} apparmor.d/`

Comment (Besanon) [reply apparmor.d/groups/lxqt/lxqt-config-powermanagement]: ah, ok changed that

File: apparmor.d/groups/lxqt/lxqt-config-powermanagement [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  owner /tmp/@{int} r,
+
+  @{sys}/class/backlight/ r,
+  @{sys}/devices/@{pci}/**/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/* rw,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/ r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/max_brightness r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/bl_power r,
+  @{sys}/devices/@{pci}/0000:00:02.0/drm/card@{int}/card@{int}-eDP-@{int}/intel_backlight/actual_brightness  r,
+  @{sys}/devices/@{pci}/**/**/drm/card@{int}/card@{int}-eDP-1/amdgpu_bl@{int}/* r,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-config-powermanagement:34]: That cannot be let like this:
- This is intel/amd specific
- This is specific to a display port output.
- `0000:00:02.0` is part of the pci address. This address often refers to the first GPU connected.

Have a look at the profile `xfpm-power-backlight-helper` and copy all sys rules from it. This is what you need here.

**Comment by Besanon** [comment]
hmm, dont know what the problem is with the last check

**Comment by roddhjav** [comment]
Don't worry, it does not concern your PR. The CI failed because an up to date version of Ubuntu now requires more rules related to snap. In other words, the CI did its jobs... and it is good.

**Comment by roddhjav** [comment]
Thanks, merged!

**Pull request merged**